loopable.cpp

```diff
@@ -1,25 +1,12 @@
 #include "loopable.h"
+#include "SmartManager.h"
 
 Loopable::Loopable() {
-    loopables.add(this);
+    SmartManager::addLoopable(this);
 }
 
 Loopable::Loopable(const bool enable) {
     if (enable) {
-      loopables.add(this);
+      SmartManager::addLoopable(this);
```

Comment:
if you move this in the setup(), then you know right away which one was registered/enabled.